packages/client/src/store/reducer.ts

```diff
@@ -0,0 +1,48 @@
+/**
+ * WordPress dependencies
+ */
+import { combineReducers } from '@wordpress/data';
+
+/**
+ * Internal dependencies
+ */
+import type { Ability } from '../types';
+import { RECEIVE_ABILITIES } from './constants';
+
+interface AbilitiesAction {
+	type: string;
+	abilities?: Ability[];
+}
+
+const DEFAULT_STATE: Record< string, Ability > = {};
+
+/**
+ * Reducer managing the abilities by ID.
+ *
+ * @param state  Current state.
+ * @param action Dispatched action.
+ * @return New state.
+ */
+function abilitiesById(
```

Comment:
Nit: abilities are stored by name, so that might be better to reflect it in the name of the reducer: 
```suggestion
function abilitiesByName(
```

Reply:
Renamed in https://github.com/WordPress/abilities-api/pull/60/commits/43ed397712f1b9028c3a29f67e51d4b507801b46